src/app/demos/tutorial/tutorial-two/index.html

```diff
@@ -1,8 +1,6 @@
 <stache
   layout="container">
 
-  <stache-container>
```

Comment:
When I added the `stache-container` back to this demo (locally), it made the content shrink extremely small. What happened to create this behavior? I'm worried that sites will have implemented a page in this way, and their pages will look bad when upgrading...

Reply:
Before we had the `stache-container` which was just a `max-width: 100%` style.  We were also making use of the `sky-container` class which was a bunch of fixed-widths.  I removed the sky class dependency, and just updated our container class to use the widths we wanted.  The `stache-container` class and component are not documented and are for internal stache use only I believe.